packages/shipjs/src/step/setup/askQuestions.js

```diff
@@ -258,3 +278,11 @@ function stringArrayValidator(answer) {
     return errorMessage;
   }
 }
+
+function isScopedPackage(name) {
```

Comment:
I think we can do this simply without `npm-package-arg`.
According to their implementation, what we need to do is to check 
- if the name starts with `@`
- and if there's `/` in it.
https://github.com/npm/npm-package-arg/blob/latest/npa.js#L124